https://github.com/nss-dev/nss/commit/edf5cb12af8a4668997b7edb65c7add4a1390b09
https://bugs.gentoo.org/849005

From: Sergei Trofimovich <slyich@gmail.com>
Date: Thu, 26 May 2022 08:08:39 +0000
Subject: [PATCH] Bug 1771273 - cpputil/databuffer.h: add missing <cstdint>
 include r=nss-reviewers,mt

Without the change build fails on this week's gcc-13 snapshot as:

    ../../cpputil/databuffer.h:20:20: error: 'uint8_t' does not name a type
       20 |   DataBuffer(const uint8_t* d, size_t l) : data_(nullptr), len_(0) {
          |                    ^~~~~~~
    ../../cpputil/databuffer.h:14:1: note: 'uint8_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
       13 | #include <iostream>
      +++ |+#include <cstdint>
       14 |

Differential Revision: https://phabricator.services.mozilla.com/D147404

--HG--
extra : moz-landing-system : lando
--- a/cpputil/databuffer.h
+++ b/cpputil/databuffer.h
@@ -11,6 +11,7 @@
 #include <cstring>
 #include <iomanip>
 #include <iostream>
+#include <cstdint>
 
 namespace nss_test {
 

